common/api/frontend-tiles.api.md

```diff
@@ -80,27 +80,18 @@ export type GraphicRepresentation = {
     format: GraphicRepresentationFormat;
     dataSource: DataSource;
 } & ({
-    status: Omit<GraphicRepresentationStatus, GraphicRepresentationStatus.Complete>;
+    status: Exclude<GraphicRepresentationStatus, "Complete">;
     url?: string;
 } | {
-    status: GraphicRepresentationStatus.Complete;
+    status: "Complete";
     url: string;
 });
 
 // @beta
-export type GraphicRepresentationFormat = "IMDL" | "3DTILES" | string;
+export type GraphicRepresentationFormat = "3DFT" | "3DTiles" | "CESIUM" | "IMODEL";
```

Comment:
Why did you remove `| string`? Are we never going to add support for a new format?

Reply:
It seemed pointless to have a type that includes some string literals but also could be any string, so I thought it would be better to enforce that the string must be one of the values currently returned by the mesh export service.

But you're right that new formats could be added at any time, and waiting until a new major version to update this type could be a problem considering the MES is still in development/tech preview.

Reply:
@eringram take a look at https://stackoverflow.com/questions/61047551/typescript-union-of-string-and-string-literals

Im kinda in favor of removing the `| string` because its a super type and pretty much invalidates any type safety we have

Reply:
Maybe since `| string` is pointless for type safety but we want to keep this open to new formats, I can remove GraphicRepresentationFormat as a type completely and just have the `format` property be a string. We can just put the list of current formats like "IMODEL", "3DTiles", etc. in the `format` prop documentation to be clear what values it should have.

I don't think this is a problem because as far as I know GraphicRepresentationFormat is not used outside this file, and if it is this will be in 5.0 with the changelog so people can react to it. 

I feel like ideally the implementation would be similar to the MapLayerFormat class that is extensible, but so far there's no need for a whole class with properties and methods in this case.

Reply:
Right. The string union exists for documentation of currently supported formats, the "| string" is there for forward compatibility.

Reply:
Well if youre adding support for a new format, wouldn't it be "widening" the type, therefore not breaking? I would think narrowing a scope (eg removing a format) is the only time it would be considered breaking. But now as I write this, I'm guessing when you mean forward compatibility that also includes removing "old" formats

Reply:
```
function doSomethingCool(graphicFormat: GraphicRepresentationFormat): string {
  switch (graphicFormat) {
    case "3DFT":
    case "3DTiles":
    case "CESIUM":
    case "IMODEL":
      return "cool";
  }

  // unreachable code becomes reachable once GraphicRepresentationFormat is expanded,
  // suddenly my function implicitly returns `undefined` and I crash at run-time.
}

Reply:
Added `| string` back to the type so it can remain extendable but the existing formats are documented. 
Also I saw this is done in a couple other places ([Tileset3dSchema.ts](https://github.com/iTwin/itwinjs-core/blob/master/core/common/src/tile/Tileset3dSchema.ts#L50), [GltfSchema.ts](https://github.com/iTwin/itwinjs-core/blob/5fc3a18aae5752681a6eef86e1ecea33bdef6ce2/core/frontend/src/common/gltf/GltfSchema.ts#L583)) so it's not completely without precedent